crates/sequencer/src/sequencer.rs

```diff
@@ -962,4 +968,31 @@ where
             .get_next_min_max_l1_fee_rate(&mut working_set)
             .map_err(|e| anyhow::anyhow!("Error reading min max l1 fee rate: {}", e))
     }
+
+    fn get_account_updates(&self) -> Result<Vec<ChangedAccount>, anyhow::Error> {
+        let head = self.db_provider.last_block()?.expect("Head must exist");
+
+        let addresses: HashSet<Address> = match head.transactions {
+            reth_rpc_types::BlockTransactions::Full(ref txs) => {
+                txs.iter().map(|tx| tx.from).collect()
```

Comment:
We already have a list of senders in `batch_receipt.tx_receipts`?

Reply:
those are for sovereign transactions